Makefile

```diff
@@ -333,6 +335,9 @@ build-windows: .prebuild prepare-enterprise
 
 build: build-linux build-windows build-osx
 
+build-current: .prebuild prepare-enterprise
```

Comment:
That's what `make dist` does. The current default. I thought the point of this was to change `make` to do what you commonly want.